### ISSUE TITLE
Provide feature flags to enable specific plugins

### ISSUE THREAD

**Issue by namit-chandwani** [aerogear/charmil#72]
Generally, feature flags allow us to activate or deactivate specific features of our application. 
In our case, we'll be allowing the SDK users to use feature flags to activate certain plugins.

This requires further investigation and discussion before starting out.

**Comment by wtrocki** [comment]
In rhoas we have something like this:
```
Flags:
  --dev-preview-commands     Enable dev preview commands
```

**Comment by wtrocki** [comment]
It will be most likely copy paste/inspired by: https://github.com/redhat-developer/app-services-cli/issues/346

**Comment by namit-chandwani** [comment]
@wtrocki Have we completely scrapped this feature or is this just out of scope for now?

**Comment by wtrocki** [comment]
We going to implement that by environment variable and it will be simple check
This was controversial from beginning :D
